### PR TITLE
cmd/server: mocked out HTTP endpoints

### PR DESCRIPTION
**Pull request by adamdecaf**
Issue: https://github.com/moov-io/ofac/issues/1

### CODE REVIEW

**Comment by adamdecaf** [comment]
cc @wadearnold / @bkmoovio for a review

File: .travis.yml [hunk 1]
```diff
@@ -36,5 +36,5 @@ script:
   - golint -set_exit_status $GOFILES
```
Comment (bkmoovio) [inline .travis.yml]: Adam, This looks great!

File: cmd/server/sdn.go [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+// GET /sdn/:id/addresses
+// - get addresses for a given SDN
+
+// GET /sdn/:id/alternateNames
+// - get alternate names for a given SDN
+
+// GET /sdn/:id
+// - get SDN information
+
```
Comment (bkmoovio) [inline cmd/server/sdn.go]: Will you be adding a mock for SDN Comments?  The data set is very small, so address and AlternateIdentity seem critical path.

File: cmd/server/companies.go [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+// POST /companies/:id/watch
+// - monitor company by id
+
+// POST /companies/watch?name=...
+// - monitor company by name, reparsed on each search
+
+// GET /companies/:id
+// - get company information and matches
+
+// POST /companies/:id
```
[the remark this answers is not in the document]
Comment (adamdecaf) [reply cmd/server/companies.go]: Oh yea. It should be!

File: cmd/server/customers.go [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+				EntityID: "2676",
+				SDNName:  "AL ZAWAHIRI, Dr. Ayman",
+				SDNType:  "individual",
+				Program:  "SDGT] [SDT",
+				Title:    "Operational and Military Leader of JIHAD GROUP",
+				Remarks:  "DOB 19 Jun 1951; POB Giza, Egypt; Passport 1084010 (Egypt); alt. Passport 19820215; Operational and Military Leader of JIHAD GROUP.",
+			},
+			Addresses: []*ofac.Address{
+				{
+					EntityID:                    "173",
```
[the remark this answers is not in the document]
Comment (adamdecaf) [reply cmd/server/customers.go]: Yea, It makes more sense to use entries that actually link to each other.

File: cmd/server/sdn.go (hunk 2 above)
Comment (adamdecaf) [reply cmd/server/sdn.go]: I wasn't planning on it.

**Pull request merged**